iModelCore/Bentley/Bentley/WString.cpp

```diff
@@ -374,6 +374,55 @@ Utf8StringR Utf8String::Trim ()
     return *this;
     }
 
+// we do not need the defaults and this is recommended post ICU>49 (we're on 64 as of writing)
+// https://unicode-org.github.io/icu/userguide/icu4c/build.html#recommended-build-options
+#define U_NO_DEFAULT_INCLUDE_UTF_HEADERS 1
+#include <unicode/utypes.h>
+#include <unicode/utf8.h>
+#include <unicode/uchar.h>
+
+//---------------------------------------------------------------------------------------
+// @bsimethod
+//---------------------------------------------------------------------------------------
+Utf8StringR Utf8String::TrimUtf8 ()

```

Comment:
move open brace to end of previous line: `() {

Reply:
done